_posts/2019-11-07-snowplow-javascript-tracker-2.12.0-released/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.12.0 released"
+title-short: Snowplow JavaScript Tracker 2.12.0
+tags: [snowplow, javascript, micro, tracker]
+author: Paul
+category: Releases
+permalink: /blog/2019/11/07/snowplow-javascript-tracker-2.12.0-released/
+discourse: true
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker]. [Version 2.12.0][2.12.0-tag] introduces a brand new test suite powered by Snowplow Micro, serving as an example of how Snowplow Micro can be integrated into a testing environment and to give us even more confidence as we continue to release updates to the Snowplow JavaScript Tracker.
+
+NB. This release continues with the change we introduced with v2.11.0 to how assets are hosted - rather than hosting the tracker on CloudFront, we now publish the asset to the GitHub release. Users who have previously relied on the CloudFront hosted asset must host the tracker on their own CDN, as is recommended practice. More detail can be found in our [v2.11.0 release post][js-tracker-2.11.0-post].
+
+Read on below the fold for:
+
+1. [New test suite powered by Snowplow Micro](#micro-tests)
+2. [Local Storage improvements](#local-storage)
+3. [Updates and bug fixes](#updates)
+4. [Upgrading](#upgrade)
+5. [Documentation and help](#doc)
+
+<!--more-->
+
+<h2 id="micro-tests">1. New test suite powered by Snowplow Micro</h2>
+
+This release introduces a new test suite into the Snowplow JavaScript Tracker which is powered by Snowplow Micro. 
+
+Snowplow Micro is a very small Snowplow Pipeline which can be initialised before running tests and then used to validate that data collection has occurred correctly. You can read more about Snowplow Micro in our earlier [blog post][snowplow-micro-post].
```

Comment:
Can you extend this with the notion that if the test fails, the build fails? I think the link between tests and stopping builds is an important one.